### PR TITLE
Add more detailed info on how to use the internal registry to mirror the images

### PR DESCRIPTION
**Pull request by fedepaol**

### CODE REVIEW

**Comment by SchSeba** [comment]
@fedepaol I think you can override this one `ci/prow/e2e-gcp` this is a documentation only PR

**Comment by fedepaol** [comment]
@SchSeba yes but I saw suspectful failures in performance. Want to check them again, and merging this is not ssuper critical.
If it fails again I will override

File: cnf-tests/README.md [hunk 1]
```diff
@@ -69,12 +69,20 @@ There are two images used by the tests that can be changed using the following e
...
 ```bash
 docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig -e CNF_TESTS_IMAGE="custom-cnf-tests-image:latests" quay.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
 ```
 
+## Instruct the tests to consume those images from a custom registry
+
+This is done by setting the IMAGE_REGISTRY environment variable:
+
+```bash
+    docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig -e IMAGE_REGISTRY=my.local.registry:5000/ -e CNF_TESTS_IMAGE="custom-cnf-tests-image:latests" quay.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
```
Comment (SchSeba) [inline cnf-tests/README.md:83]: To be consistent can you change the image registry to this `IMAGE_REGISTRY="my.local.registry:5000/"`

File: cnf-tests/README.md [hunk 2]
```diff
@@ -121,12 +129,86 @@ The following command
...
+        "image": "imagefordpdk:4.5"
+    }
+]
+```
+
+And by passing it to the mirror command, for example saving it locally as `images.json`.
+With the following command, the local path is mounted in `/kubeconfig` inside the container and that can be passed to the mirror command.
+
+```bash
+    docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig quay.io/openshift-kni/cnf-tests /usr/bin/mirror -registry my.local.registry:5000/ -images /kubeconfig/images.json |  oc image mirror -f -
```
Comment (SchSeba) [inline cnf-tests/README.md:156]: -registry my.local.registry:5000/ -> --registry "my.local.registry:5000/"

Comment (SchSeba) [reply cnf-tests/README.md:156]: -images /kubeconfig/images.json -> --images /kubeconfig/images.json

File: cnf-tests/README.md [hunk 3]
```diff
@@ -121,12 +129,86 @@ The following command
...
+oc create ns cnftests
+```
+
+Make that imagestream available to all the namespaces used for tests. This is required to allow the tests namespaces to fetch the images from the cnftests imagestream.
+
+```bash
+oc policy add-role-to-user system:image-puller system:serviceaccount:sctptest:default --namespace=cnftests
+oc policy add-role-to-user system:image-puller system:serviceaccount:cnf-features-testing:default --namespace=cnftests
+oc policy add-role-to-user system:image-puller system:serviceaccount:performance-addon-operators-testing:default --namespace=cnftests
+oc policy add-role-to-user system:image-puller system:serviceaccount:dpdk-testing:default --namespace=cnftests
```
Comment (SchSeba) [inline cnf-tests/README.md:185]: please also add the "dpdk" namespace not only the "dpdk-testing" one

**Comment by fedepaol** [comment]
Overridden as this is doc only

**Pull request merged**